test/plugins/features/movies/controller.test.js

```diff
@@ -23,14 +43,67 @@ describe('movie controller', () => {
 
   });
 
-  describe('get', () => {
+  describe('list', () => {
+
+    it('list movies released after year_start', () => {
+      const payload = { year_start: 1983 };
+
+      return Controller.list(payload)
+      .then((json) => {
```

Comment:
Nit: this is a collection, not really JSON, so it might be better to call this variable `movies`.